lib/grizzly/command_class/association_group_info/group_command_list_get.ex

```diff
@@ -0,0 +1,98 @@
+defmodule Grizzly.CommandClass.AssociationGroupInfo.GroupCommandListGet do
+  @moduledoc """
+  Command module for working the Association Group Info command class ASSOCIATION_COMMAND_LIST_GET command
+
+  Command Options:
+
+    * `:seq_number` - the sequence number used by the Z/IP packet
+    * `:retries` - the number of attempts to send the command (default 2)
+    * `:group` - the group id
+  """
+  @behaviour Grizzly.Command
+
+  alias Grizzly.Packet
+  alias Grizzly.Command.{EncodeError, Encoding}
+  alias Grizzly.CommandClass.AssociationGroupInfo
+
+  @type t :: %__MODULE__{}
+
+  @type opt ::
+          {:seq_number, Grizzly.seq_number()}
+          | {:retries, non_neg_integer()}
+          | {:group, non_neg_integer()}
+
+  defstruct seq_number: nil, retries: 2, group: nil
+
+  @spec init([opt]) :: {:ok, t}
+  def init(opts) do
+    {:ok, struct(__MODULE__, opts)}
+  end
+
+  @spec encode(t) :: {:ok, binary} | {:error, EncodeError.t()}
+  def encode(%__MODULE__{seq_number: seq_number, group: group} = command) do
+    allow_cache = 0x01
+
+    with {:ok, _encoded} <-
+           Encoding.encode_and_validate_args(command, %{
+             group: :byte
+           }) do
+      binary =
+        Packet.header(seq_number) <> <<0x59, 0x05, allow_cache::size(1), 0x00::size(7), group>>
```

Comment:
I think that `0x80` is the same thing you're accomplishing here with he bit-sized numbers? If so, maybe we can use `0x80` just to make the code a little more concise and beginner-friendly?

Reply:
Not sure about this. The way it is now maps directly to the documentation. Collapsing the 0X01::size(1) for allow_cache and 0X00::size(7) would in fact be obfuscating.

Reply:
fair point :+1: